force_wfmanager/plugin_dialog.py

```diff
@@ -0,0 +1,119 @@
+from __future__ import unicode_literals
+from traitsui.editors import HTMLEditor
+from traitsui.list_str_adapter import ListStrAdapter
+
+from force_bdss.api import BaseExtensionPlugin
+from traits.api import (
+    HasStrictTraits, List, Instance, Unicode,
+    on_trait_change
+)
+from traitsui.api import View, UItem, ListStrEditor, HGroup
+
+
+class PluginAdapter(ListStrAdapter):
+    def _get_text(self):
+        return self.item.name if self.item.name else self.item.id
+
+    def _get_text_color(self):
+        if self.item.broken:
+            return "#FF0000"
+        return super(PluginAdapter, self)._get_text_color()
+
+
+class PluginDialog(HasStrictTraits):
+    """Displays a list of the available plugins, together with their
+    description.
+    """
+    plugins = List(Instance(BaseExtensionPlugin))
+
+    selected_plugin = Instance(BaseExtensionPlugin)
+
+    selected_plugin_HTML = Unicode()
```

Comment:
I was going to suggest this should be HTML(), but looking into it HTML has the parent class of String (rather than BaseUnicode).